### DISCUSSION TITLE
How to share content within team with or without use of App_Data?

### DISCUSSION THREAD

**Discussion by borysl** [OrchardCMS/OrchardCore#11018] [Q&A]
Hello everyone,

We have started developing website 2 month ago and taken OrchardCore as foundation. There are more content management than coding, but coding also exists. The database is in PostgreSQL and supported by usual EF migrations.
After switching to .Net 6 and version 1.2.1 we unable to keep App_Data (with tenant.json, ContentDefinition.json and media files) in the project anymore and when Content root path is Website project (and not bin/Debug/net6.0) the website [fails to start](https://i.ibb.co/gPxJz6P/2022-01-12-22h00-48.png)

For us it means that we cannot run project from VS anymore (publishing still works).

Question: is there is a way to maintain both content AND code in version control system shared in the team. We considered recipies, but at least in version 1.0.0 it didn't work.

This is how page look like:
```
Generated Code
One or more compilation references may be missing. If you're seeing this in a published application, set 'CopyRefAssembliesToPublishDirectory' to true in your project file to ensure files in the refs directory are published.

The type or namespace name 'Microsoft' could not be found in the global namespace (are you missing an assembly reference?)
+
[assembly: global::Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemAttribute(typeof(AspNetCore._Areas_OrchardCore_Contents_Views_Item_Display), @"mvc.1.0.view", @"/Areas/OrchardCore.Contents/Views/Item/Display.cshtml")]
Predefined type 'System.Type' is not defined or imported
+
[assembly: global::Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemAttribute(typeof(AspNetCore._Areas_OrchardCore_Contents_Views_Item_Display), @"mvc.1.0.view", @"/Areas/OrchardCore.Contents/Views/Item/Display.cshtml")]
Predefined type 'System.String' is not defined or imported
+
[assembly: global::Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemAttribute(typeof(AspNetCore._Areas_OrchardCore_Contents_Views_Item_Display), @"mvc.1.0.view", @"/Areas/OrchardCore.Contents/Views/Item/Display.cshtml")]
Predefined type 'System.String' is not defined or imported
+
[assembly: global::Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemAttribute(typeof(AspNetCore._Areas_OrchardCore_Contents_Views_Item_Display), @"mvc.1.0.view", @"/Areas/OrchardCore.Contents/Views/Item/Display.cshtml")]
The type or namespace name 'System' could not be found (are you missing a using directive or an assembly reference?)
```

**Comment by borysl** [comment]
After some investigation I understood that ContentRootPath has nothing with it - in VS it is the same as if I successfully run it from console

**Comment by borysl** [comment]
In fact it is when I use ASPNETCORE_ENVIRONMENT=Production it works when ASPNETCORE_ENVIRONMENT=Development it doesn't work.

**Comment by syampolsky** [comment]
@borysl , did you figure this issue out? I am having the same problem.

**Reply by testazuretest1** [reply]
@syampolsky were you able to figure it out?

**Comment by Skrypt** [comment]
Looks like a custom module that is referencing an older version of the OrchardCore.Contents module. Make sure you have updated all your assembly references in your custom modules to match the version you want to use. Older version of the OrchardCore.Contents module may not be able to find newly globally referenced assemblies. This is what it seems to fail to find here "System, System.String" ...
